### PR TITLE
Add CONTRIBUTING.rst file

### PR DESCRIPTION
**Pull request by AntonioND**
I guess that it doesn't hurt to have a guide for this. I'll leave it here for a few days before merging it.

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -0,0 +1,88 @@
+Contributing
+============
+
+RGBDS was created in the late 90's and has received contributions of several
+developers since then. It wouldn't have been possible to get to this point
+without their work and. For that reason, it is always open to the contributions
```
Comment (Ben10do) [inline CONTRIBUTING.rst]: “...without their work, and for that reason, it...”

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+If you think you have a good idea that should be implemented, you can propose it
+to be done.
+
+1. Create a new `issue <https://github.com/rednex/rgbds/issues>`__.
+
+2. Try to be as accurate as possible. Describe what you need and why you need
+   it, maybe with examples.
+
+You need to understand that the contributors are doing it in their free time, so
```
Comment (Ben10do) [inline CONTRIBUTING.rst]: This feels a little forceful to me, how about “Please understand that...” or “Please remember that...”?

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -0,0 +1,88 @@
+Contributing
+============
+
+RGBDS was created in the late 90's and has received contributions of several
```
Comment (Ben10do) [inline CONTRIBUTING.rst]: “...received contributions **from** several developers...”

File: CONTRIBUTING.rst [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+4. Create a new `issue <https://github.com/rednex/rgbds/issues>`__.
+
+Of course, it may not always be possible to give an accurate bug report, but it
+always helps to fix it.
+
+Requesting new features
+-----------------------
+
+If you think you have a good idea that should be implemented, you can propose it
```
Comment (Ben10do) [inline CONTRIBUTING.rst]: This feels like it’s questioning the contributor’s idea somewhat... How about “If you come up with a good idea that could be implemented, ...”?

File: CONTRIBUTING.rst (hunk 3 above)
Comment (AntonioND) [reply CONTRIBUTING.rst]: Done

File: CONTRIBUTING.rst (hunk 1 above)
Comment (AntonioND) [reply CONTRIBUTING.rst]: “...without their work and, for that reason, it...” :P

File: CONTRIBUTING.rst (hunk 4 above)
Comment (AntonioND) [reply CONTRIBUTING.rst]: Done

File: CONTRIBUTING.rst (hunk 2 above)
Comment (AntonioND) [reply CONTRIBUTING.rst]: Done

**Comment by AntonioND** [comment]
Also, I guess it's time for another release... I'll try to get it ready over the next days.

**Pull request closed without merging**